docs/source/related_topics/troubleshooting/troubleshooting.md

```diff
@@ -1,3 +1,45 @@
+```python
```

Comment:
install troubleshooting -> this
quirks -> this
os-specific stuff -> individual pages

Reply:
virtualenv stuff changed